Formula/packer.rb

```diff
@@ -4,8 +4,8 @@ class Packer < Formula
   desc "Tool for creating identical machine images for multiple platforms"
   homepage "https://packer.io"
   url "https://github.com/mitchellh/packer.git",
```

Comment:
As pointed in the comments, this line should probably become `url "https://github.com/hashicorp/packer.git"`